belogging/filters.py

```diff
@@ -64,10 +67,10 @@ def filter(self, record):
 
         if len(self._cache) >= self._cache_size:
             # oldest key with less hits
```

Comment:
in the cache access above (L61), it should not have a lock too ?

Reply:
I guess... yes.. I'll have to put the whole if block inside the lock..

Reply:
done.